wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -941,15 +933,12 @@ public void onTrimMemory(final int level) {
         }
     }
 
-    private Notification createNetworkSyncNotification(BlockchainState blockchainState) {
+    private Notification createNetworkSyncNotification() {
         Intent notificationIntent = OnboardingActivity.createIntent(this);
         PendingIntent pendingIntent = PendingIntent.getActivity(this, 0,
                 notificationIntent, PendingIntent.FLAG_UPDATE_CURRENT);
 
-        String message = BlockchainStateUtils.getSyncStateString(blockchainState, this);
-        if (message == null) {
-            message = getString(R.string.blockchain_state_progress_downloading);
-        }
+        String message = getString(R.string.blockchain_state_progress_downloading);
```

Comment:
I would propose this code, so that the notification can be updated with the current state.  But the first time it is called in startForeground, then `blockchainState = null`

```java
        String message = blockchainState != null ? BlockchainStateUtils.getSyncStateString(blockchainState, this) : null;
        if (message == null) {
            message = getString(R.string.blockchain_state_progress_downloading);
        }
```